### DISCUSSION TITLE
Audio Video

### DISCUSSION THREAD

**Discussion by brixxnatt** [ant-media/Ant-Media-Server#6445] [Q&A]
Ability to add seperate audio for the incoming video to  endpoint

**Comment by mekya** [comment]
Hi @brixxnatt,

I don't understand what exactly you want to do. Could you please explain further?

Cheers
Oz

**Comment by brixxnatt** [comment]
if I have a stream coming to the ant media server , and i want to republish the stream via endpoint , i need ability to change the audio of the stream , and put seperate audio like mp3 , m3u8 , etc as audio to the restreaming video

**Comment by burak-58** [comment]
You may do this using Filter Plugin. You may check this blogpost:
https://antmedia.io/discover-the-filter-plugin/

In general, you can create 2 streams one for video and one for audio and usinf filter plugin you may merge them into one output stream. Let me convert this to a discussion. You may get more help in discussions.
